vllm_ascend/attention/attention.py

```diff
@@ -135,7 +135,7 @@ def get_splitfuse_attn_mask(
                       context_len:] = self.splitfuse_mask_value
             right_tensor = attn_mask[current_row:current_row + q_len,
                                      context_len:seq_len]
-            right_tensor.mask_fill_(
```

Comment:
Thanks for the fix. Can you add a e2e test for get_splitfuse_attn_mask usage case?  i.e. add a test with chunked prefill  enable? This kind of error is  stupid. lol